Dockerfile.proto

```diff
@@ -0,0 +1,66 @@
+
+ARG UBUNTU_VERSION=22.04
+ARG TFSERVING_VERSION=2.9.0
+
+# Build image, genearates proto files
+FROM arm64v8/ubuntu:${UBUNTU_VERSION}  AS build-image
```

Comment:
make sure platform is correctly selected in the workflow to avoid this warning

Reply:
I'll sync this change with Angelo to verify on which branch to do it. I.e. Not fixed yet...